flask_api/static/page_data/story/storybooks/storybooks.md

```diff
@@ -1 +0,0 @@
-# Storybooks
```

Comment:
You get the title for free and it's okay to have a yaml page defined with no markdown so I just removed the stubs which produced duplicate titles.